SHAREDLIBS_INSTALL_INC_DIR not set

Still trying to package libuv on nixos
```
-- Lua: using information from luarocks
-- LUA_LIBDIR: 
-- LUA_INCDIR: /nix/store/491m3i5n0l92aar68hjy0zv61yw11jc6-luajit-2.1.0-beta3/include/luajit-2.1
-- LUA: /nix/store/491m3i5n0l92aar68hjy0zv61yw11jc6-luajit-2.1.0-beta3/bin/luajit
-- Lua library: LUA_LIBRARIES-NOTFOUND
-- path 
CMake Error at CMakeLists.txt:253 (install):
  install FILES given no DESTINATION!
```
Going through the various steps, `SHAREDLIBS_INSTALL_INC_DIR` ,  I find `set(SHAREDLIBS_INSTALL_INC_DIR "${INSTALL_INC_DIR}")` but INSTALL_INC_DIR is possibly never set (hence the error), whereas   INSTALL_LIB_DIR defaults to ${LIBDIR}. Would be cool to have a nice default.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -202,23 +202,27 @@ if (NOT LUA)
   if (BUILD_STATIC_LIBS)
     set(STATICLIBS_INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib"
       CACHE PATH "Installation directory for static libraries")
-    set(STATICLIBS_INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include/luv"
-      CACHE PATH "Installation directory for headers")
   endif (BUILD_STATIC_LIBS)
   if (BUILD_SHARED_LIBS)
     set(SHAREDLIBS_INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib"
       CACHE PATH "Installation directory for shared libraries")
-    set(SHAREDLIBS_INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include/luv"
-      CACHE PATH "Installation directory for headers")
   endif (BUILD_SHARED_LIBS)
 else ()
   # use paths from luaRocks
   set(MODULE_INSTALL_LIB_DIR "${INSTALL_LIB_DIR}")
   set(STATICLIBS_INSTALL_LIB_DIR "${INSTALL_LIB_DIR}")
-  set(STATICLIBS_INSTALL_INC_DIR "${INSTALL_INC_DIR}")
   set(SHAREDLIBS_INSTALL_LIB_DIR "${INSTALL_LIB_DIR}")
-  set(SHAREDLIBS_INSTALL_INC_DIR "${INSTALL_INC_DIR}")
 endif ()
+
+# header install paths are LuaRocks-agnostic, so just use CMAKE_INSTALL_PREFIX regardless
+if (BUILD_STATIC_LIBS)
+  set(STATICLIBS_INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include/luv"
+    CACHE PATH "Installation directory for headers")
+endif (BUILD_STATIC_LIBS)
+if (BUILD_SHARED_LIBS)
+  set(SHAREDLIBS_INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include/luv"
+    CACHE PATH "Installation directory for headers")
+endif (BUILD_SHARED_LIBS)
 
 if (CMAKE_INSTALL_PREFIX)
   if (BUILD_MODULE)

